### PR TITLE
Testdata tf: crop2d model with test data added.

### PR DESCRIPTION
**Pull request by ashishkrshrivastava**
Relating PR: https://github.com/opencv/opencv/pull/16473

### CODE REVIEW

**Comment by dkurt** [comment]
Looks like changes in `generate_tf_model` are missed

**Comment by ashishkrshrivastava** [comment]
@dkurt changes in `generate_tf_model` has been made along with addition of crop2d_net.pbtxt file.

File: testdata/dnn/tensorflow/crop2d_net.pbtxt [hunk 1]
```diff
@@ -0,0 +1,268 @@
+node {
```
Comment (dkurt) [inline testdata/dnn/tensorflow/crop2d_net.pbtxt]: This file is unused in tests. Please remove.

Comment (ashishkrshrivastava) [reply testdata/dnn/tensorflow/crop2d_net.pbtxt]: Sure.

**Comment by dkurt** [comment]
Please replace also `writeBlob(outputData, name + '_out')` to `writeBlob(np.ascontiguousarray(outputData), name + '_out')` and regenerate the data. Looks like the tests are failed because of it.

**Comment by ashishkrshrivastava** [comment]
> Please replace also `writeBlob(outputData, name + '_out')` to `writeBlob(np.ascontiguousarray(outputData), name + '_out')` and regenerate `crop2d_out.npy`. Looks like the tests are failed because of it.

@dkurt , Build successfully without using np.ascontiguousarray. Do you still want me to do this ?

**Pull request merged**